api_v3/services/ReportService.php

```diff
@@ -440,21 +440,13 @@ public function getCsvFromStringParamsAction($id, $params = null, $excludedField
 	public function exportToCsvAction(KalturaReportExportParams $params)
 	{
 		$this->validateReportExportParams($params);
-
+		$kuser = kCurrentContext::getCurrentKsKuser();
+		$partner = PartnerPeer::retrieveByPK(kCurrentContext::getCurrentPartnerId());
 		if (!$params->recipientEmail)
 		{
-			$kuser = kCurrentContext::getCurrentKsKuser();
-			if ($kuser)
-			{
-				$params->recipientEmail = $kuser->getEmail();
-			}
-			else
-			{
-				$partnerId = kCurrentContext::getCurrentPartnerId();
-				$partner = PartnerPeer::retrieveByPK($partnerId);
-				$params->recipientEmail = $partner->getAdminEmail();
-			}
+			$params->recipientEmail = $kuser ? $kuser->getEmail() : $partner->getAdminEmail();
 		}
+		$params->recipientName = $kuser ? $kuser->getFullName() : $partner->getName();
```

Comment:
If you already exported this param to be set in the api call, then why are you overriding it? why not check also if it was set, do not override it (like in the email)

Reply:
The api doesn't have a field for name, only for email.

Reply:
updated the code.